setup.py

```diff
@@ -16,11 +16,15 @@
 # limitations under the License.
 ################################################################################
 
+import os
+
 from setuptools import setup, find_packages
 
 
-with open('README.rst') as readme:
-    long_description = readme.read()
+here = os.path.abspath(os.path.dirname(__file__))
```

Comment:
Why abspath?

Reply:
Ask @srid, the co-maintainer at [python-gnureadline](https://github.com/ludwigschwardt/python-gnureadline). I lifted it from its `setup.py` and deposited it freely everywhere :-)

Reply:
Maybe it makes more sense the other way around, as found on various forums?

```Python
here = os.path.dirname(os.path.abspath(__file__))
```

Reply:
Still not clear to me why you'd need the abspath at all.

Reply:
On the other hand I guess it probably doesn't hurt, so feel free to keep it if it's going to be a pain to remove. FYI spead2 skips the abspath: https://github.com/ska-sa/spead2/blob/1e8f7d6f6ed16036f3e999ae00b8c17efba29663/setup.py#L132